packages/mendel-resolver/variational-resolver.js

```diff
@@ -1,32 +1,51 @@
 const ModuleResolver = require('./index');
 const path = require('path');
+const variationMatches = require('mendel-development/variation-matches');
 
 class VariationalModuleResolver extends ModuleResolver {
-    constructor({projectRoot, envNames, basedir, baseVariationDir, variationDirs, variations}) {
+    constructor({
+        envNames,
+        variations,
+        // entry related
+        variation,
+        normalizedId,
+        basedir,
+        // config params
+        projectRoot,
+        baseConfig,
+        variationConfig,
+    }) {
         super({cwd: projectRoot, basedir, envNames});
 
-        // Must be a path relative to the basedir
+        this.variationChain = variations.map(variationPath => {
+            return path.resolve(projectRoot, variationPath);
+        });
+        this.baseVarDir = path.resolve(projectRoot, baseConfig.dir);
+
+        this.variation = variation;
+        this.normalizedId = normalizedId;
```

Comment:
disregarding the fact that this property is not used, how will this be used for this package? resolving, in my view, has nothing to do with normalizedId. Intentional?